Kernel/Syscalls/prctl.cpp

```diff
@@ -14,14 +14,34 @@ ErrorOr<FlatPtr> Process::sys$prctl(int option, FlatPtr arg1, [[maybe_unused]] F
     VERIFY_PROCESS_BIG_LOCK_ACQUIRED(this);
     return with_mutable_protected_data([&](auto& protected_data) -> ErrorOr<FlatPtr> {
         switch (option) {
-        case PR_GET_DUMPABLE:
+        case PR_GET_DUMPABLE: {
             return protected_data.dumpable;
-        case PR_SET_DUMPABLE:
+        }
+        case PR_SET_DUMPABLE: {
             if (arg1 != 0 && arg1 != 1)
                 return EINVAL;
             protected_data.dumpable = arg1;
             return 0;
         }
```

Comment:
I don't think these curly braces are needed; the rule we have for multiple if/else branches needing curlies doesn't spply to switch statements :)